reemgawad/aircams
# Add actions to rental summary page

What has changed

- As an Owner You cannot rent a camera if it's yours
- As an Owner you can see all you rental request from "my camera" ( aka requests you'd need to approve)
- As an Owner you can approve or decline a request
- As a renter you can cancel your request
- As a renter you can access you rentals summary details from "my rentals"

How to test
As an owner you should not be able to rent a camera
As a renter Create a rental and try to cancel it 
As an owner update status of your rentals ( that need your approval (under My_cameras)
